contrib/clojure-ui/www/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
```

Comment:
Is the folder `www` a convention in clojure world? If it is not then can we call this one public instead?

Reply:
by [6c18531](https://github.com/bentoml/OpenLLM/pull/89/commits/6c185318def6bec519ff6b089ee157160d67a62a)